ArduCopter/Copter.h

```diff
@@ -439,10 +408,11 @@ class Copter : public AP_HAL::HAL::Callbacks {
     // Attitude, Position and Waypoint navigation objects
     // To-Do: move inertial nav up or other navigation variables down here
 #if FRAME_CONFIG == HELI_FRAME
-    AC_AttitudeControl_Heli attitude_control;
+    #define AC_AttitudeControl_t AC_AttitudeControl_Heli
 #else
-    AC_AttitudeControl_Multi attitude_control;
+    #define AC_AttitudeControl_t AC_AttitudeControl_Multi
 #endif
+    AC_AttitudeControl_t attitude_control;
```

Comment:
since you are at it... wouldn't it be feasible to just allocate the right AttitudeControl in the constructor rather having the ifdef